app/celery/tasks.py

```diff
@@ -122,27 +127,35 @@ def send_sms(self,
         return
 
     try:
-        dao_create_notification(
-            Notification.from_api_request(
-                created_at, notification, notification_id, service.id, SMS_TYPE, api_key_id, key_type
```

Comment:
Yes it is not being used now. I think I have to create a new method without the notificaiton_id, start using the new method, deploy that, then remove the old method. I was going to worry about it in another PR.